types/types.go

```diff
@@ -120,6 +133,9 @@ type BlobInfo struct {
 	// MIME type for compressed layers (e.g., gzip or zstd). This field MUST be
 	// set when `CompressionOperation == Compress`.
 	CompressionAlgorithm *compression.Algorithm
+	// CryptoOperation is used in Image.UpdateLayerInfos to instruct
+	// whether the original layer was encrypted/decrypted
+	CryptoOperation LayerCrypto
```

Comment:
This really needs cleaning up; “edits to blob” are not “properties of a blob”, and should be separated / only maintained for `ManifestUpdateOptions`.

(Yes, compression added the parameters here.)